ui-tests/cypress/integration/users/groups.spec.js

```diff
@@ -66,7 +66,6 @@ describe('Groups', () => {
     currentPage.getDialog().getBody().getStateInfo().should('contain', groupCode);
 
     currentPage.getDialog().confirm();
-    cy.reload(); //TODO the page does not automatically refresh the table
```

Comment:
Is it now refreshing it?

Reply:
Yes, the table automatically updates now when a group is deleted, and keeping the `cy.reload()` actually makes the test fail